QgisModelBaker/libqgsprojectgen/dataobjects/layers.py

```diff
@@ -167,21 +164,32 @@ def post_generate(self, project):
 
             self.__form.add_element(tab)
 
-            relations_to_add = set()
+            relations_to_add = []
```

Comment:
Thanks for this elegant solution. Out of curiosity, what has been your thought about not using a set anymore and make a list instead?

Reply:
Just to keep ordering constant over multiple generates, because set is unordered and could potentially get to different tab order each time.